app/src/Login/ResetPswPage.tsx

```diff
@@ -0,0 +1,292 @@
+import React, {useState} from 'react';
```

Comment:
A lot of this code looks the same as in Signup and maybe Login. Can you factor out the same stuff and put it into the folder utils and then reference that code.